### ISSUE TITLE
Incorrect apt source on Ubuntu 18.04

### ISSUE THREAD

**Issue by lschweiss-wustl** [deric/puppet-beegfs#15]
With beegfs::release: '7.1' the apt source is incorrectly set to:

```
# This file is managed by Puppet. DO NOT EDIT.
# beegfs
deb [arch=amd64] http://www.beegfs.io/release/beegfs_7_1 bionic non-free

It should be remapped to:
deb [arch=amd64] http://www.beegfs.io/release/beegfs_7_1 deb9 non-free
```

**Comment by deric** [comment]
@lschweiss-wustl Thanks for reporting the issue. It's a bit more complicated, Ubuntu 18.04 is in fact based on Debian Buster, the appropriate release would be Debian 10. But BeeGFS currently has only releases for `jessie` and `stretch`, so `stretch` (or `deb9`) would be the closest match.

**Comment by deric** [comment]
Should be fixed in [v0.5.0 release](https://github.com/deric/puppet-beegfs/releases/tag/v0.5.0).
